lib/utils/tests-options-validator.js

```diff
@@ -48,7 +48,49 @@ Object.defineProperty(TestsOptionsValidator.prototype, 'shouldSplit', {
  */
 function validatePartitions(partitions, split) {
   validatePartitionSplit(partitions, split);
-  validatePartitionsUnique(partitions);
+  validateElementsUnique(partitions);
+}
+
+/**
+ * Validates the specified reply-browser
+ *
+ * @param {*} replayBrowser
+ * @param {*} replayExecution
+ */
+function validateReplyBrowser(replayBrowser, replayExecution) {
+  if(!replayExecution) {
+    throw new Error('You must specifiy a \'reply-execution\' value in order to use \'reply-browser\' option.');
+  }
+
+  for (const i in replayBrowser) {
+    const browserId = replayBrowser[i]
```

Comment:
I would just do 
if (replayBrowser[i] < 1) 
no need to create a separate constant if you're not going to use it for anything.

Reply:
One more thing to validate should be browserId > browserNumbers
where browserNumbers is recorded in the json file